src/tag.rs

```diff
@@ -1,7 +1,7 @@
 use crate::frame::Content;
```

Comment:
Could you please include some unit test coverage for reading and writing? You can do this by updating the `make_tag` function in `src/stream/tag.rs`.

Bonus points if you have an actual file from a different program than rust-id3 so we can ensure compatibility

Reply:
> Bonus points if you have an actual file from a different program than rust-id3 so we can ensure compatibility

Is there some convenient tool to dump the raw data from the file?

Previously I used `eyeD3 --write-objects`, but that only dumps the object data, not the whole frame.

Reply:
Eh, I just used `dd` in the past. The MPEG data starts at `0xff 0xfx` so everything before that is the ID3 tag.

Reply:
> Eh, I just used `dd` in the past. The MPEG data starts at `0xff 0xfx` so everything before that is the ID3 tag.

Maybe I misunderstood. Do you mean 0xFF 0xFx == 0xFFF0 == 65520 ?
Because running this works, but if I play back the file using `mpv` I can hear the first 1.5 seconds of the track:

    dd if=my_file.mp3 of=geob.id3 bs=65520 count=1

So the byte count is wrong.

Reply:
Anyway, I just parsed the header by hand:

    $ head -c 10 music.mp3 | hexdump -C
    00000000  49 44 33 03 00 00 00 02  16 61                    |ID3......a|
    0000000a
    $ python
    Python 3.9.1 (default, Dec 13 2020, 11:55:53)
    [GCC 10.2.0] on linux
    Type "help", "copyright", "credits" or "license" for more information.
    >>> (0x61 | 0x16 << 7 | 0x02 << 14) + 10
    35691
    >>>
    $ dd if=music.mp3 of=geob.id3 bs=35691 count=1
    1+0 records in
    1+0 records out
    35691 bytes (36 kB, 35 KiB) copied, 0.000232432 s, 154 MB/s

Maybe it would be a good idea to expose the functionality to retrieve the header and add a `tagdump` tool to the `examples/` directory.

Reply:
Added real-world GEOB data in https://github.com/polyfloyd/rust-id3/pull/55/commits/05365a6f63ccc29a1ebb1bfa9fe082e253da9f88